clang/test/Driver/sycl-spirv-ext-old-model.c

```diff
@@ -129,3 +186,110 @@
 // CHECK-CPU-SAME:,+SPV_KHR_non_semantic_info
 // CHECK-CPU-SAME:,+SPV_KHR_cooperative_matrix
 // CHECK-CPU-SAME:,+SPV_INTEL_fp_max_error"
```

Comment:
```suggestion
// CHECK-CPU-FPME:,+SPV_INTEL_fp_max_error"
```
To match the suggested change with `-check-prefixes` above.